src/GraphODataTemplateWriter/CodeHelpers/TypeScript/TypeScriptNamespace.cs

```diff
@@ -35,9 +36,18 @@ public class TypeScriptNamespace
         // constants
         private const int MaxLineLength = 120;
         private const string MainNamespaceName = "Microsoft.Graph";
-        private const string TypeScriptMainNamespaceName = "microsoftgraph";
+        private string TypeScriptMainNamespaceName = "microsoftgraph" + (ConfigurationService.Settings.Properties != null && ConfigurationService.Settings.Properties.ContainsKey("typescript.namespacePostfix") ? ConfigurationService.Settings.Properties["typescript.namespacePostfix"] : string.Empty);
         private const string TabSpace = "    ";
 
+        /// <summary>
+        /// Returns the main or top level namespace
+        /// </summary>
+        /// <returns></returns>
+        public string GetMainNameSpace()
```

Comment:
nit: namespace is considered a single word.
```suggestion
        public string GetMainNamespace()
```

Reply:
Made the correction.